packages/visx-demo/happo/gallery.tsx

```diff
@@ -13,17 +10,7 @@ type HappoSnapshot = {
   };
 };
 
-const specialCases = new Set(['@visx/demo-axis', '@visx/demo-xychart']);
-
-// renders an example with a timeout
-const renderWithTimeout: (
-  Example: React.ReactElement,
-) => HappoSnapshot['variants'][string] = Example => renderInDom => {
-  return new Promise(resolve => {
-    renderInDom(Example);
-    setTimeout(resolve, asyncTimeout);
-  });
-};
+const specialCases = new Set([]);
```

Comment:
Any reason to keep this around?

Reply:
initially I thought it'd still be useful for testing specific examples quickly. removed.